rosbag2_converter_default_plugins/src/rosbag2_converter_default_plugins/cdr/cdr_converter.cpp

```diff
@@ -63,37 +63,50 @@ std::string get_package_library_path(const std::string & package_name)
 CdrConverter::CdrConverter()
 {
   auto library_path = get_package_library_path("rmw_fastrtps_cpp");
-  std::shared_ptr<Poco::SharedLibrary> library;
-  try {
-    library = std::make_shared<Poco::SharedLibrary>(library_path);
-  } catch (Poco::LibraryLoadException &) {
-    throw std::runtime_error(
-            std::string("poco exception: library could not be found:") + library_path);
+
+  std::unique_ptr<rcutils_shared_library_t> library =
+    std::make_unique<rcutils_shared_library_t>();
+  *library = rcutils_get_zero_initialized_shared_library();
```

Comment:
Is it possible to move this to the constructor instead?

Reply:
Maybe it makes sense to abstract this on rcpputils in a class to be able to create an destroy the object. What do you think @wjwwood ?

Reply:
Move it to the constructor of which class?

You could add a companion api to rcpputils, but I don’t think that’s necessary unless we think lots of  places in the code will be using this in conjunction with cpp.

Reply:
Could that call be joined though?
```c++
auto library = std::make_unique<rcutils_shared_library_t>(rcutils_get_zero_initialized_shared_library());
```